src/fsipy/automatedPostprocessing/postprocessing_h5py/create_spectrograms_chromagrams.py

```diff
@@ -0,0 +1,232 @@
+    if power_scaled:
+        composite_figure_name += "_power_scaled"
+
+    composite_figure_path = Path(image_folder) / (composite_figure_name + '.png')
+    fig1.savefig(composite_figure_path)
+
+    # create separate spectrogram figure
+    fig2, ax2_1 = plt.subplots()
+    fig2.set_size_inches(7.5, 5)
+    title = f"Pxx max = {max_val:.2e}, Pxx min = {min_val:.2e}, threshold Pxx = {lower_thresh}"
+    spec.plot_spectrogram(fig2, ax2_1, bins, freqs, Pxx, ylim, title=title,
+                          x_label="Time (s)", color_range=[thresh_val, max_plot])
+
+    # Save data to files (spectrogram)
+    spec_filename = f"{dvp}_{case_name}_{num_windows}_windows_thresh{thresh_val}_spectrogram"
+    path_to_spec = Path(image_folder) / (spec_filename + '.png')
+    fig2.savefig(path_to_spec)
+
+    output_csv_path = path_to_spec.with_suffix('.csv')
+    data_csv = np.append(freqs[np.newaxis].T, Pxx, axis=1)
+    bins_txt = np.array2string(bins, max_line_width=10000, precision=2, separator=',').replace("[", "").replace("]", "")
+    np.savetxt(output_csv_path, data_csv, header=bins_txt, delimiter=",")
+
+    # Save data to files (chromagram)
+    chroma_filename = f"{dvp}_{case_name}_{num_windows}_windows_chromagram"
+    path_to_chroma = Path(image_folder) / (chroma_filename + '.png')
+    chroma_y = np.linspace(0, 1, chroma.shape[0])
+
+    output_csv_path = path_to_chroma.with_suffix('.csv')
+    data_csv = np.append(chroma_y[np.newaxis].T, chroma, axis=1)
+    bins_txt = \
+        np.array2string(bins_raw, max_line_width=10000, precision=2, separator=',').replace("[", "").replace("]", "")
+    np.savetxt(output_csv_path, data_csv, header=bins_txt, delimiter=",")
+
+    # Save data to files (SBI)
+    sbi_filename = f"{dvp}_{case_name}_{num_windows}_windows_SBI"
+    path_to_sbi = Path(image_folder) / (sbi_filename + '.png')
+
+    csv_output_path = path_to_sbi.with_suffix('.csv')
+    data_csv = np.array([bins, chroma_entropy]).T
+    np.savetxt(csv_output_path, data_csv, header="t (s), SBI", delimiter=",")
+
+
+def main():
+    # Load in case-specific parameters
+    args = spec.read_command_line_spec()
+
+    # Create logger and set log level
+    logging.basicConfig(level=args.log_level, format="%(message)s")
+
+    # Create or read in spectrogram dataframe
+    dvp, df, case_name, image_folder, visualization_hi_pass_folder = \
+        spec.read_spectrogram_data(args.folder, args.mesh_path, args.save_deg, args.stride, args.start_time,
+                                   args.end_time, args.n_samples, args.ylim, args.sampling_region,
+                                   args.fluid_sampling_domain_id, args.solid_sampling_domain_id, args.r_sphere,
+                                   args.x_sphere, args.y_sphere, args.z_sphere, args.dvp, args.interface_only,
+                                   args.component, args.point_id, sampling_method=args.sampling_method)
```

Comment:
Would it be possible to remove some of the command-line argument and use value from `default_variables.json` instead?